src/common/template.test.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { expect } from "chai";
+import { Template } from "./template";
+import { fileSync } from "tmp";
+import { writeFileSync, readFileSync } from "fs-extra";
```

Comment:
In my PR, I've added the `chai-fs` plugin. You should consider using it, I've found it quite helpful.